R/tm_front_page.R

```diff
@@ -15,17 +15,17 @@
 #' element, if named the name is shown first in bold, followed by the value.
 #' @param show_metadata (`logical`) indicating whether the metadata of the datasets be available on the module.
 #'
-#' @examples
+#' @return A `teal` module to be used in `teal` applications.
```

Comment:
Is there any way we can use the `@return` from shared_params? https://github.com/insightsengineering/teal.modules.general/pull/670/files#diff-60bef5af43a1bd541299ce9bceeb226afb23deaa5e637b145c85e05ffd552b0cR28

@averissimo would we need to write `@inheritSection Value` in here?

Reply:
I actually think this is missing in all modules

Reply:
`@inheritSection Value` (or `@inheritSection return`)

Reply:
`#' @inherit shared_params return` should do it

Reply:
```suggestion
#' @inherit shared_params return
```